apps/demo/config/vite/app.ts

```diff
@@ -65,6 +65,7 @@ export default defineConfig({
           join(pathRelativeToRepositoryRoot, 'node_modules'),
           join(pathRelativeToRepositoryRoot, 'node_modules/@lmc-eu/spirit-design-tokens/src/scss'),
         ],
+        silenceDeprecations: ['mixed-decls'],
```

Comment:
**suggestion (if-minor):** Can you please add a comment to this like `// @see https://sass-lang.com/documentation/breaking-changes/mixed-decls/` so we have the reference what does it solves?

Maybe also mention that we are silencing false-positives...?

Reply:
added